ethcore/trace/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+description = "Transaction tracing"
+name = "trace"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+common-types = { path = "../types"}
+ethcore-blockchain = { path = "../blockchain" }
+ethcore-db = { path = "../db" }
+ethereum-types = "0.6"
+evm = { path = "../evm" }
```

Comment:
Again requiring entire EVM seems odd, would be worth to remove it in the future.

Reply:
We use `CallType` from the `evm`, that's all. Seems like a good opportunity for a refactor indeed.